config/default.py

```diff
@@ -7,8 +7,7 @@
 
 SECRET_KEY = 'thisisnearlyimpossible'
 
-WTF_CSRF_ENABLED = True
-WTF_CSRF_SECRET_KEY = 'you-will-never-guess'
+WTF_CSRF_ENABLED = False
```

Comment:
Because Backbone can't use WTForms, so for now I disabled CSRF. I added an issue to add CSRF back in the future, but we'll probably have to use something like this: http://danialk.github.io/blog/2013/07/28/advanced-security-in-backbone-application/